documentation/modules/upgrading/proc-upgrade-cli-tool-files.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cli-tool-files-{context}']
+= Converting custom resources configuration files using the API conversion tool
+
+[role="_abstract"]
+This procedure describes how to use the API conversion tool to convert YAML files describing the configuration for Strimzi custom resources into a format applicable to `{ApiVersion}`.
+To do so, you use the `convert-file` (`cf`) command.
+
+The command can convert YAML files containing multiple documents.
+For a multi-document YAML file, all the Strimzi custom resources it contains are converted.
+Any non-Strimzi Kubernetes resources are replicated unmodified in the converted output file.
+
+After you have converted the YAML file, you can apply the configuration to update the custom resource in the cluster.
+Alternatively, if the GitOps synchronization mechanism is being used for updates on your cluster, you can use it to apply the changes.
```

Comment:
Can we make it more clear here that the conversion is really finished only once the resource is converted in the Kube cluster? To make sure nobody converts the YAML and things it is all good.

Reply:
Added: _The conversion is only complete when the custom resource is updated in the Kubernetes cluster. _